### ISSUE TITLE
Difficulty recognizing Genesis 1:1 John 1:1

### ISSUE THREAD

**Issue by jeremyam** [openbibleinfo/Bible-Passage-Reference-Parser#59]
When attempting to parse two passages where the second passage is only separated by a single space and contains a book name that is also found in multiple books, i.e. John is found 1 John, 2 John, and 3 John etc., current parser finds a bc Genesis 1 and bcv 1 John 1:1.

**Comment by alerque** [comment]
Both for the sake of any human readers and the parser sequential references like this should be separated (e.g. with a coma or semicolon) to disambiguate the potential meanings.

**Comment by jeremyam** [comment]
I agree that references like these _should_ be written in comma-separated sequences. However, there is a deficiency since the book numbers of 1, 2, or 3 John should never be attached to a previous reference via a colon like this. Any human reader would not see Genesis 1:1 John 1:1 and think the verse of Genesis 1:1 belonged to the next reference. And given the use case I'm currently working on, the references are not separated by any punctuation marks.

**Comment by openbibleinfo** [comment]
@jeremyam This behavior reflected an intentional decision to bind to books more tightly than to chapter:verse indicators. The Twitter data from the repo suggests that in this case, however, it might be better to bind to the colon instead of the book. I changed this behavior in v3 for books that start with a number.
